python/pyspark/tests/test_rdd.py

```diff
@@ -669,6 +670,12 @@ def test_sample(self):
         wr_s21 = rdd.sample(True, 0.4, 21).collect()
         self.assertNotEqual(set(wr_s11), set(wr_s21))
 
+    def test_datetime(self):
```

Comment:
Should probably add a comment like:

```
SPARK-39176: ... 
```

See also https://spark.apache.org/contributing.html

Reply:
It has been added and modified, please approve it again.